test/metabase/driver/sql_jdbc/sync/describe_table_test.clj

```diff
@@ -126,8 +149,31 @@
            {:fk-column-name "VENUE_ID", :dest-table {:name "VENUES", :schema "PUBLIC"}, :dest-column-name "ID"}}
          (sql-jdbc.describe-table/describe-table-fks :h2 (mt/id) {:name "CHECKINS"}))))
 
+(deftest describe-fields-or-table-test
```

Comment:
I didn't find any test coverage for `:database-position` for the default SQL-JDBC implementation so I added this test. It also sense checks a few other things, like sensible base types and names.